app/errors.py

```diff
@@ -0,0 +1,18 @@
+class VaccineError(Exception):
+    def __init__(self, message: str = "Default VaccineError message") -> None:
+        super().__init__(message)
+
+
+class NotVaccinatedError(VaccineError):
+    def __init__(self, message: str = "Default VaccineError message") -> None:
+        super().__init__(message)
+
+
+class OutdatedVaccineError(VaccineError):
+    def __init__(self, message: str = "Default VaccineError message") -> None:

```

Comment:
The default message for the `OutdatedVaccineError` class should be more descriptive and specific to the error it represents. Right now, it's the same as the `VaccineError` class, which can be misleading. Please consider changing it to something like 'The vaccine is outdated.'.